package.json

```diff
@@ -16,17 +16,17 @@
         "dev:build": "npm run dev && teeny build",
         "dev:develop": "npm run dev && teeny develop",
         "dev:init": "npm run dev && teeny init",
-        "dev": "rm teeny-* || true && npm pack && npm i -g teeny-*.tgz -f",
+        "dev": "sudo npm uninstall -g teeny-cli && (rm teeny-* || true) && npm pack && sudo npm i -g teeny-*.tgz",
```

Comment:
Hmm maybe we can add another `dev:` script?

Uninstalling the global `teeny-cli` feels like too much for this dev script to do - even though I see why you've added it here